### ISSUE TITLE
Can't locate object method "inflate" via package "Math::Polynomial"

### ISSUE THREAD

**Issue by eserte** [mhasch/perl-Math-Polynomial-Cyclotomic#1]
On some of my smoker systems the test suite fails like this:
```
Can't locate object method "inflate" via package "Math::Polynomial" at /home/cpansand/.cpan/build/2021101720/Math-Polynomial-Cyclotomic-0.003-3/blib/lib/Math/Polynomial/Cyclotomic.pm line 65.
# Looks like your test exited with 255 just after 9.
t/01_basics.t ........ 
Dubious, test returned 255 (wstat 65280, 0xff00)
Failed 10/19 subtests 
Can't locate object method "inflate" via package "Math::Polynomial" at /home/cpansand/.cpan/build/2021101720/Math-Polynomial-Cyclotomic-0.003-3/blib/lib/Math/Polynomial/Cyclotomic.pm line 65.
# Looks like your test exited with 255 just after 8.
t/02_memoized.t ...... 
Dubious, test returned 255 (wstat 65280, 0xff00)
Failed 9/17 subtests 
Can't locate object method "unnest" via package "Math::Polynomial" at /home/cpansand/.cpan/build/2021101720/Math-Polynomial-Cyclotomic-0.003-3/blib/lib/Math/Polynomial/Cyclotomic.pm line 191.
# Looks like your test exited with 255 just after 10.
t/03_aurifeuille.t ... 
Dubious, test returned 255 (wstat 65280, 0xff00)
Failed 48/58 subtests 
```

Statistical analysis suggests that this happens if an older version of Math::Polynomial is installed (1.018 or older):
```
****************************************************************
Regression 'mod:Math::Polynomial'
****************************************************************
Name           	       Theta	      StdErr	 T-stat
[0='const']    	      0.0000	      0.0000	   0.00
[1='eq_1.017'] 	      0.0000	      0.0000	   0.00
[2='eq_1.018'] 	      0.0000	      0.0000	   0.00
[3='eq_1.019'] 	      1.0000	      0.0000	35420643986635940.00
[4='eq_1.021'] 	      1.0000	      0.0000	38996845566727088.00

R^2= 1.000, N= 49, K= 5
****************************************************************
```

**Comment by mhasch** [comment]
Thank you, Slaven, for your very precise report.
Indeed the Math::Polynomial v1.014 dependency given in Makefile.PL is not sufficient,
as we use features added in v1.018 and v1.019.
I'll adjust both the metadata and the actual "use" statements.

-Martin

**Comment by mhasch** [comment]
Fixed in Math-Polynomial-Cyclotomic-0.004.
